client/src/app/+accounts/accounts.component.html

```diff
@@ -10,8 +10,15 @@
           <div class="actor-name">{{ account.nameWithHost }}</div>
 
           <span *ngIf="user?.blocked" [ngbTooltip]="user.blockedReason" class="badge badge-danger" i18n>Banned</span>
-
-          <my-user-moderation-dropdown buttonSize="small" [user]="user" (userChanged)="onUserChanged()" (userDeleted)="onUserDeleted()">
+          <span *ngIf="account.mutedByUser" class="badge badge-danger" i18n>Muted</span>
+          <span *ngIf="account.mutedServerByUser" class="badge badge-danger" i18n>Muted by your instance</span>
+          <span *ngIf="account.mutedByInstance" class="badge badge-danger" i18n>Instance muted</span>
+          <span *ngIf="account.mutedServerByInstance" class="badge badge-danger" i18n>Instance muted by your instance</span>
```

Comment:
We could use [dropdown headers](https://getbootstrap.com/docs/4.0/components/dropdowns/#menu-headers) to separate actions specific to the user from moderation actions that apply instance-wide.

Reply:
https://github.com/Chocobozzz/PeerTube/issues/1290